lib/rexml/parsers/baseparser.rb

```diff
@@ -505,15 +505,13 @@ def pull_event
       private :pull_event
 
       def entity( reference, entities )
-        value = nil
```

Comment:
Could you revert this change for readability?
It shows that the default `value` is `nil` explicitly.

Reply:
Okay, I reverted the change in [b5da207](https://github.com/ruby/rexml/pull/199/commits/b5da2078bc7e7c6331ed3f7d8369a052d172c5fe)